test/libsolidity/syntaxTests/parsing/location_specifiers_for_params.sol

```diff
@@ -0,0 +1,5 @@
+contract Foo {
+    function f(uint[] storage constant x, uint[] memory y) internal { }
```

Comment:
`storage constant` is invalid (`Storage location has to be "memory" (or unspecified) for constants.`). Should we leave it as an error or use `constant` rather on the `memory` argument?